src/main/java/com/max/grpc/orders/client/rest/services/MenuServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+
+package com.max.grpc.orders.client.rest.services;
+
+import com.max.grpc.orders.client.CafeClient;
+import com.max.grpc.orders.client.rest.models.ApiCafeMenu;
+import com.max.grpc.orders.client.rest.models.ApiFoodItem;
+import com.max.grpc.orders.proto.CafeMenu;
+
+import org.apache.log4j.Logger;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Path("/menu")
+public class MenuServiceImpl implements MenuService {

```

Comment:
This is Endpoint, Api ... whatever but Service. You need a Service or at least a mapper to do the job. The endpoint should receive that Service and operate with it. It should not know about gRPC, proto or any other details of the implementation

Reply:
Done